main/tests.py

```diff
@@ -64,8 +65,15 @@ def test_root_entrypoint_redirection_logged_daycare_user(self, client, create_da
         assert response['Location'] == '/homepage/'
 
 
+@pytest.mark.django_db
 class TestHomepageView:
     def test_unlogged_user_access_to_homepage(self, client):
         response = client.get("/homepage/")
         assert response.status_code == 302
         assert response['Location'] == '/login/?next=/homepage/'
+
+    def test_dog_owner_homepage_is_visible_for_dog_owner(self, client, create_dog_owner_user):
+        client.force_login(user=create_dog_owner_user.user)
+        response = client.get("/homepage/")
+        assert response.status_code == 200
+        assert list(response.context['daycares']) == list(DayCare.objects.all())
```

Comment:
I want this PR merged ASAP, so please open a ticket to: assign both sides of the assertion equation to variables, then assert between the variables. it will improve the readability of the test.

Reply:
Opened #104